include/bluetooth/gatt.h

```diff
@@ -706,10 +706,18 @@ struct _bt_gatt_ccc {
 	/** Highest value of all connected peer's subscriptions */
 	uint16_t value;
 
-	/** @brief CCC attribute changed callback
+	/** @brief React to presence of subscribers
 	 *
-	 *  @param attr   The attribute that's changed value
-	 *  @param value  New value
+	 *  This event signals whether the Bluetooth stack has recipients for
+	 *  bt_gatt_indicate() or bt_gatt_notify() for @p attr with @p conn set to
+	 *  @c NULL.
+	 *
+	 *  This signal starts out as "not present" after boot.
+	 *
+	 *  Note: This event is not sequenced in relation to connection events.
+	 *
+	 *  @param attr The attribute the presence information belongs to.
+	 *  @param value If 0, then "not present". Otherwise "present".
```

Comment:
Please refer to the `BT_GATT_CCC_NOTIFY` and `BT_GATT_CCC_INDICATE` values

Reply:
I just noticed that this value is broken at the moment if lazy CCC loading is not enabled. The reduction includes bonded not connected peers.

Reply:
> I think documenting this is over-specifying the meaning of value and we should give the weaker guarantee.

I don't think it is over-specifying to specify the possible values that this function may be called with. IMO it would be a clear improvement to do this.